exporters/otlp/test/otlp_exporter_test.cc

```diff
@@ -1,4 +1,5 @@
 #include "opentelemetry/exporters/otlp/otlp_exporter.h"
+#include "opentelemetry/context/threadlocal_context.h"
```

Comment:
nit: move this to be the first line?

Reply:
Moved.

Reply:
Although when I run the formatter it reorders the includes back to the original way.